oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -157,6 +161,10 @@ public class MongoDocumentStore implements DocumentStore {
      */
     public static final long DEFAULT_THROTTLING_TIME_MS = Long.getLong("oak.mongo.throttlingTime", 20);
 
+    /**
+     * Document size of 16MB is a limit in Mongo
+     */
+    private static final long SIZE_LIMIT = 16777216;
```

Comment:
Can you please remove these lines. `SIZE_LIMIT` is not used.

Reply:
@stefan-egli with one of my commits, had added this back.